cfme/v2v/conversion_hosts.py

```diff
@@ -0,0 +1,238 @@
+
+    next_btn = Button("Next")
+    configure_btn = Button("Configure")
+    close_btn = Button("Close")
+
+    @property
+    def is_displayed(self):
+        return self.title.text == "Configure Conversion Host"
+
+    @View.nested
+    class location(View):  # noqa
+        provider_type = BootstrapSelect("providerType")
+        provider_name = BootstrapSelect("provider")
+        cluster = BootstrapSelect("cluster")
+
+        @property
+        def is_displayed(self):
+            return self.provider_type.is_displayed
+
+        def after_fill(self, was_change):
+            self.parent.next_btn.click()
+
+    @View.nested
+    class hosts(View):  # noqa
+        hostname = SearchBox(id='host-selection')
+
+        @property
+        def is_displayed(self):
+            return self.hostname.is_displayed
+
+        def after_fill(self, was_change):
+            self.parent.next_btn.click()
+
+    @View.nested
+    class authentication(View):  # noqa
+        # conversion_host_ssh_key = TextInput(locator='.//textarea[@id="host-ssh-key-input"]')
+        conversion_host_ssh_key = HiddenFileInput(locator='.//input[contains(@type,"file")]')
+        transformation_method = BootstrapSelect("transformationMethod")
+        vddk_library_path = Input(id='vddk-library-path')
+        vmware_ssh_private_key = TextInput(locator='.//textarea[@id="vmware-ssh-key-input"]')
+        tls_cert_switch = V2VBootstrapSwitch(label='Verify TLS Certificates for OpenStack')
+        osp_ca_cert = TextInput(locator='.//textarea[@id="openstack-ca-certs-input"]')
+
+        @property
+        def is_displayed(self):
+            return self.conversion_host_ssh_key.is_displayed
+
+        def after_fill(self, was_change):
+            import time
+            time.sleep(10)
+            self.parent.configure_btn.click()
+
+    @View.nested
+    class results(View):  # noqa
+        msg = Text('.//h3[contains(@class,"blank-slate-pf-main-action")]')
+        close_btn = Button("Close")
+
+        @property
+        def is_displayed(self):
+            return self.msg.is_displayed
+
+@attr.s
+class ConversionHost(BaseEntity):
+    """Class representing conversion Host
+        Args:
+            target_provider: RHV or OSP
+            provider_name: rhv43 or osp13-ims
+            cluster: Default
+            hostname: host to be configured
+            conversion_host_ssh_key: /etc/pki/ovirt-engine/keys/engine_id_rsa on RHV-M.
+            transformation_method: vddk or ssh
+            vddk_library_path: url for vddk
+            vmware_ssh_private_key: /var/lib/vdsm/.ssh/id_rsa on rhv host
+            osp_ca_cert
+
+    """
+    target_provider = attr.ib()
+    provider_name = attr.ib()
+    cluster = attr.ib()
+    hostname = attr.ib()
+    conversion_host_ssh_key = attr.ib()
+    transformation_method = attr.ib()
+    vddk_library_path = attr.ib(default=None)
+    vmware_ssh_private_key = attr.ib(default=None)
+    tls_cert_switch = attr.ib(default=None)
```

Comment:
Don't take this as a switch value explicitly, if value `osp_ca_cert` is present it will be `True` or it will be `False`, what say?